compiler/wasm/src/index.cts

```diff
@@ -0,0 +1,44 @@
+import { FileManager } from './noir/file-manager/file-manager';
+import { createNodejsFileManager } from './noir/file-manager/nodejs-file-manager';
+import { NoirWasmCompiler } from './noir/noir-wasm-compiler';
+import { LogData, LogFn } from './utils';
+import { CompiledCircuit } from './types/noir_artifact';
+
+async function compile(fileManager: FileManager, projectPath?: string, logFn?: LogFn, debugLogFn?: LogFn) {
```

Comment:
Can you define the return type here? We should be explicit on our interface.

Reply:
Done!